transformations/tests/test_pool_allocator.py

```diff
@@ -275,19 +297,15 @@ def test_pool_allocator_temporaries(frontend, generate_driver_stack, block_dim,
     calls = FindNodes(CallStatement).visit(driver.body)
     assert len(calls) == 1
     if nclv_param:
-        if check_bounds:
-            expected_args = ('1', 'nlon', 'nlon', 'nz', 'field1(:,b)', 'field2(:,:,b)', 'ylstack_l', 'ylstack_u')
-            assert calls[0].arguments == expected_args
-        else:
-            expected_args = ('1', 'nlon', 'nlon', 'nz', 'field1(:,b)', 'field2(:,:,b)', 'ylstack_l')
-            assert calls[0].arguments == expected_args
+        expected_args = ('1', 'nlon', 'nlon', 'nz', 'field1(:,b)', 'field2(:,:,b)')
     else:
-        if check_bounds:
-            expected_args = ('1', 'nlon', 'nlon', 'nz', '2', 'field1(:,b)', 'field2(:,:,b)', 'ylstack_l', 'ylstack_u')
-            assert calls[0].arguments == expected_args
-        else:
-            expected_args = ('1', 'nlon', 'nlon', 'nz', '2', 'field1(:,b)', 'field2(:,:,b)', 'ylstack_l')
-            assert calls[0].arguments == expected_args
+        expected_args = ('1', 'nlon', 'nlon', 'nz', '2', 'field1(:,b)', 'field2(:,:,b)')
+    if check_bounds:
+        expected_kwargs = (('YDSTACK_L', 'ylstack_l'), ('YDSTACK_U', 'ylstack_u'))
```

Comment:
minor thing: not always consistent with upper and lower case ...

Reply:
Yes, the reason for this is subtle: The original code is all upper case, but our symbols are case-insensitive, so whenever we're doing symbol comparison the original test style of lower case is fine. The one exception to that are the keywords when using keyword arguments, which are treated as pure (case sensitive) strings. Changing this might be a good idea in general, but quite invasive, so I'd like to keep this for a separate PR.